src/app/views/staff/staff.controller.coffee

```diff
@@ -1,20 +1,44 @@
-@App.controller 'StaffController', ($filter, $stateParams, $uibModal, toastr, MnoeUsers, MnoAppsInstances) ->
+@App.controller 'StaffController', ($log, $stateParams, $window, $uibModal, toastr, MnoeCurrentUser, MnoeUsers, MnoeSubTenants, ADMIN_ROLES) ->
   'ngInject'
   vm = this
+  vm.isSaving = false
+  vm.adminRoles = ADMIN_ROLES
 
-  vm.staff =
-    # Display staff creation modal
-    createModal: ->
-      modalInstance = $uibModal.open(
-        templateUrl: 'app/views/staff/create-staff-modal/create-staff.html'
-        controller: 'CreateStaffController'
-        controllerAs: 'vm'
-      )
+  vm.isAdmin = MnoeCurrentUser.user.admin_role == 'admin'
```

Comment:
You should maybe call `MnoeCurrentUser.getUser()` before accessing its properties. If the user refreshes the page it may end up to an error.